### ISSUE TITLE
Move SetupWindow as a Welcome Screen inside the app like other elementary apps

### ISSUE THREAD

**Issue by Suzie97** [PhotostatEditor/Photostat#10]
At the moment, we show the setup window before loading the main window like pixelmator pro. And then ask the user to either open a new project, or import existing photos etc. 

![Screenshot from 2021-06-19 14-44-44](https://user-images.githubusercontent.com/68198116/122637578-23c7aa80-d10d-11eb-9d3e-c0e90ed1ce57.png)

﻿I've been reading the docs lately and thinking about this, the reason we decided to implement an extra window at the first place was purely for UI purposes. I've never felt very happy with the impact of this decision on the UX of the app. 

If we incorporate the welcome view provided by elementary inside the app window, it will feel like the app is fast, the initial barrier to get to work will be decreased as less setup will be required. 

This will also be more consistent with other elementary apps. Also, theoretically, it should give us some performance boost as less resources will be loaded. Users who'll be regularly using the app would also find this extra window annoying because they know the layout of the app and how it works.

And in the future, if the elementary team decides to change the welcome view, we get it for free without tweaking our existing code.

@syfds, @Abdallah-Moh please share your thoughts on this.

**Comment by Abdallah-Moh** [comment]
I think that the window is much user friendly stacking every thing in one window will be a mess

**Comment by Suzie97** [comment]
> I think that the window is much user friendly stacking every thing in one window will be a mess

I don't think so, because the welcome screen will only be displayed when a current project is not open. So when nothing is open, there will be no layers, no properties nothing, just the tools sidebar and the headerbar.

**Comment by colinkiama** [comment]
> I think that the window is much user friendly stacking every thing in one window will be a mess

If this is ever becomes a problem then we coded it wrong in my opinion.

Single window makes more sense to me. We could copy how the elementary devs have implemented their welcome screens.

**Comment by Suzie97** [comment]
@colinkiama , @Abdallah-Moh As we decided to implement a single window approach, now we need to decide if we should expose the ToolsSidebar, Layers Panel and Properties Panel along with the welcome screen in the middle or if we should use display the welcome widget in the middle grid along with the panels and tools?

As far as prior art goes, elementary apps do expose other elements of the MainWindow UI along with the welcome screen.

![Screenshot from 2021-07-10 07-15-44](https://user-images.githubusercontent.com/68198116/125148282-bbd61400-e14e-11eb-9ce8-8cc5fec57711.png)
![Screenshot from 2021-07-10 07-14-46](https://user-images.githubusercontent.com/68198116/125148284-be386e00-e14e-11eb-96d4-d5d9c1907a50.png)

**Comment by Abdallah-Moh** [comment]
@Suzie97 We should not

**Comment by colinkiama** [comment]
Yeah there's no point 🤷‍♂️
